backend/__init__.py

```diff
@@ -12,7 +12,9 @@
 from flask_principal import PermissionDenied
 from jsonschema import FormatChecker, Draft4Validator
 from pymongo import MongoClient, ReturnDocument
-from mongoengine.fields import UUIDField, ListField, StringField, BooleanField
+from mongoengine.fields import UUIDField, ListField, StringField, BooleanField, ObjectId
+from mongoengine.fields import EmbeddedDocumentField, EmbeddedDocument, ListField, \
+    ObjectIdField
```

Comment:
It's still two imports from mongoengine.fields now. Also pep8 recommends braces, so 

```
from mongoengine.fields import (UUIDField, ListField, StringField, BooleanField, ObjectId
EmbeddedDocumentField, EmbeddedDocument, ListField, ReferenceField)
```